src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -3,6 +3,20 @@
 export RUNTIME_PATH=$1
 export EXECUTION_DIR=$(dirname "$0")
 
+function print_info_from_core_file_using_lldb {
+  local core_file_name=$1
+  local executable_name=$2
+  local plugin_path_name="$RUNTIME_PATH/shared/Microsoft.NETCore.App/9.9.9/libsosplugin.so"
+
+  # check for existence of lldb-3.9 on the path
+  hash lldb-3.9 2>/dev/null || { echo >&2 "lldb-3.9 was not found. Unable to print core file."; return; }
```

Comment:
There was some discussion in the issue about potentially using newer lldb versions
https://github.com/dotnet/core-eng/issues/2880#issuecomment-375480404
Can you check that ALL the distros will have 3.9 ? If not you will have to try other versions as well

Reply:
SOS plugin containing commands like ```clrstac``` and ```pe``` works for versions greateer than >=3.8

Reply:
please test the failure path here if yo udidn't

Reply:
> SOS plugin containing commands like clrstac and pe works for versions greateer than >=3.8

I thought it was 3.9, but anyway, my point was _newer_ vesrions. If the machine doesn't have 3.9 but does have 5.0, this won't find it. Is 3.9 always there? If not we should look for the others.